### ISSUE TITLE
Support request: A line or two about how and why to set GO15VENDOREXPERIMENT

### ISSUE THREAD

**Issue by samuell** [derekparker/delve#321]
Would be useful to have this info right there in the "how to build" docs, rather than needing to google around.

**Comment by nkev** [comment]
Agree. Also, nowhere in the OSX install section does it tell you how and where to install delve itself! Stepping through the code only works intermittently on my MBPR Yosemite... shame.

**Comment by derekparker** [comment]
This information what actually already on the wiki [here](https://github.com/derekparker/delve/wiki/Building) but I went ahead and updated it with details and links on how to opt into Go 1.5 vendor experiment and what it is. I also made it more clear in the actual step by step installation instructions, adding it to the command line examples.

@nkev Not sure what you mean here, the OS X section contains the information for installing the self signed cert as well as the `make` command used to install Delve. Also, there is work being done to help improve the experience on OSX, definitely stay tuned and send in any issues you happen to come across.

**Comment by nkev** [comment]
Sorry, I may be doing misunderstanding the steps as a newby, but in the third last OSX bullet point:

* [At least on Yosemite:] In keychains select category Keys -> dlv-cert -> right click -> GetInfo -> Access Control -> select "Allow all applications to access this item" -> Save Changes.

I could not find "Access Control" in the `Get Info` dialog box in my MBP (Yosemite), so I had to skip this step.

The last OSX bullet point:

* Run the following: CERT=dlv-cert make install, which will install the binary and codesign it.

It's not clear which `make` file you refer to so presumed I had to do a `go get https://github.com/derekparker/delve`  and CD to the delve root directory to access the make file.
